virtualenv_tools.py

```diff
@@ -255,9 +255,9 @@ def get_orig_path(venv_path: str) -> str:
     with open(activate_path) as activate:
         for line in activate:
             # virtualenv 20 changes the position
-            for possible in ('VIRTUAL_ENV="', "VIRTUAL_ENV='"):
+            for possible in ('VIRTUAL_ENV="', "VIRTUAL_ENV='", "VIRTUAL_ENV="):
                 if line.startswith(possible):
-                    return line.split(possible[-1], 2)[1]
+                    return line.split(possible[-1], 2)[1].strip()
```

Comment:
this line makes no sense now (and is broken if the path contained an `=`)

it's intent is to strip quotes so that should not be happening in the unquoted version

Reply:
specifically: the code that writes this uses `shlex.quote` -- this should probably use `shlex.split` to parse this -- something like

```python
value, = shlex.split(line.removeprefix('VIRTUAL_ENV=))
```

Reply:
Oh yeah that definitely does work much better, thanks!
Switched to using `shlex` though I did have to use a split instead of `removeprefix` since that's not available yet in Python3.8